src/compat/index.ts

```diff
@@ -45,6 +45,7 @@ export { sortBy } from './array/sortBy.ts';
 export { size } from './array/size.ts';
 export { some } from './array/some.ts';
 export { zipObjectDeep } from './array/zipObjectDeep.ts';
+export { filter } from './array/filter.ts';
```

Comment:
I guess reordering exports should be done in a separate pull request 😢

Reply:
I missed that part😢 I will keep it in mind for my next contribution. Thank you for the advice!